tox.ini

```diff
@@ -138,5 +131,16 @@ deps =
     {[testenv]deps}
     pytest-cov
 
+[testenv:py3]
+basepython = {env:TOXPYTHON:python3}
+setenv =
+    {[testenv]setenv}
+usedevelop = true
+commands =
+    {posargs:pytest --cov --cov-report=term-missing -vv}
+deps =
+    {[testenv]deps}
+    pytest-cov
+
```

Comment:
This uses whatever `py3` version the user has installed?

Reply:
Yes.

Reply:
Then double approve! Very nice addition!